platform/platform-configuration-listener/pom.xml

```diff
@@ -29,6 +29,10 @@
     </description>
     <packaging>bundle</packaging>
 
+    <properties>
```

Comment:
There's already a `powermock.version` property in `ddf/pom.xml`. Can't we use that one?